MetaMask/metamask-extension
# Adding security alert setting to setting search

## **Description**
Fix the issue that PPOM security alerts are not searchable in settings.

## **Manual testing steps**

1. Go to settings page
2. Search for "Security Alerts"
3. It should show search result
4. Click the search result and it should take you to experimental page

## **Screenshots/Recordings**
Before:
![Screenshot 2023-10-03 at 6 41 02 PM](https://github.com/MetaMask/metamask-extension/assets/2182307/62f3f283-00cc-4f69-9878-c415d0e0ad04)


After:
![Screenshot 2023-09-28 at 7 42 36 PM](https://github.com/MetaMask/metamask-extension/assets/2182307/80cc2358-eb3c-4c0c-870f-27c716e2f751)

## **Related issues**

_Fixes #20753

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
